### DISCUSSION TITLE
Adding compatibility to more Linux distributions?

### DISCUSSION THREAD

**Discussion by mys31f** [OpenTabletDriver/OpenTabletDriver#1079] [General]
Would there be support for RPM-based distros? Haven't had much success self-compiling with .NET recently on Fedora 33.

**Comment by InfinityGhost** [comment]
> Would there be support for RPM-based distros? Haven't had much success self-compiling with .NET recently on Fedora 33.

An RPM package has been in high demand, we will be adding one in [v0.6](https://github.com/OpenTabletDriver/OpenTabletDriver/milestone/13).
